bindings/py/cpp_src/bindings/algorithms/py_TemporalMemory.cpp

```diff
@@ -119,18 +119,12 @@ using namespace nupic::algorithms::connections;
         ));
 
 
-        py_HTM.def("activateCells", [](HTM_t& self, py::array_t<nupic::UInt32>& activeColumns, bool learn)
+        py_HTM.def("activateCells", [](HTM_t& self, const SDR& activeColumns, bool learn)
```

Comment:
FYI: Pybind11 often ignores `const` keywords.  Python does not really have a concept of constant.